language-configuration.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I wish this could be somehow part of the LSP protocol. Then, clients would not need to implement something like this.

Reply:
I agree, and I was surprised to find out that it wasn't.